hub.json

```diff
@@ -461,6 +461,9 @@
     "windsor-ai": [
         "dbt_facebook_ads"
     ],
+    "Xebia Data": [
```

Comment:
```suggestion
    "godatadriven": [
```

The key of this dict is the GitHub username where the repo lives

Reply:
Done in 02b2be9!